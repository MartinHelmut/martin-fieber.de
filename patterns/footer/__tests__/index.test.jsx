import React from "react";
import renderer from "react-test-renderer";

import Footer from "../index";

describe("Footer", () => {
  test("should render", () => {
    const tree = renderer.create(<Footer>Bottom data</Footer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
