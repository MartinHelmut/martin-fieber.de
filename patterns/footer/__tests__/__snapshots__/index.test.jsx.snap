// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<p
  className="text__Text-sc-1ado5hh-0 footer__Footer-sc-5zdu7-0 kDVjiF"
>
  Bottom data
</p>
`;
